src/workers/notifications/campaignUpdatedAction.ts

```diff
@@ -44,9 +43,6 @@ const worker: TypedNotificationWorker<'skadi.v2.campaign-updated'> = {
       const err = originalError as TypeORMQueryFailedError;
 
       if (err?.name === 'EntityNotFoundError') {
-        // some campaigns do not exist on API so just warn in case we need to check later
-        logger.warn({ err, params }, 'could not find campaign');
-
```

Comment:
Why are we removing the logs?

The issue Chris reported was that the worker is retrying and logs have nothing to do with it.